src/lib/util/limitTextLength.test.ts

```diff
@@ -0,0 +1,7 @@
+import { limitTextLength } from './limitTextLength';
+
+describe('Limit text length testing', () => {
```

Comment:
Would you be able to use `text-overflow: ellipsis;` instead?

Reply:
That was my first implementation but `text-overflow: ellipsis` limits text to one line. There will be cases in the future where descriptions will need to partially displayed but more than one line. Do you see any potential issues with implementing it this way?

Reply:
I'm not sure but I believe `text-overflow: ellipsis` has some accessibility features behind it. @talo242 wdyt?

Reply:
Yeah, it's more accessible because all the text is in the HTML and it's only being visually hidden using CSS.
The bad thing with ellipsis is that we cannot specify how many characters we want to display, we have to rely on the container's width.

I think we can still use the current approach and add a `title` attribute to the element. That way screen readers can read the whole text and also if someone hovers over it it will display the whole text.